spec/dashboards/building_dashboard_spec.rb

```diff
@@ -1,6 +1,6 @@
 require "rails_helper"
 
-describe BuildingDashboard do
+RSpec.describe BuildingDashboard do #, type: :controller do
```

Comment:
Do we need the `#, type: :controller do`? Isn't that all commented out?

Reply:
I thought I deleted that.